flexget/plugins/input/limit.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import itertools
+import logging
+
+from flexget import plugin
+from flexget.event import event
+
+log = logging.getLogger('limit')
+
+
+class PluginInputs(object):
+    """
+    Limits the number of entries an input plugin can produce.
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'amount': {'type': 'integer', 'minimum': -1},
+            'plugin': {
```

Comment:
Change to "from" to be in line with discover etc

Reply:
Oh, good idea, that's much nicer.